apps/website/src/components/common/Contact.tsx

```diff
@@ -13,18 +13,25 @@ function Contact({ name, institution, role, link, imgSrc }: any) {
           alt="Profile Picture"
         />
       </div>
-      <div className="flex-1 px-4 text-left">
-        <H3>{name}</H3>
-        <Prose>{role}</Prose>
-        <Prose>{institution}</Prose>
-        <a
-          className="text-uzh-red-100 hover:underline"
-          target="_blank"
-          href={link}
-          rel="noreferrer"
-        >
-          Profile
-        </a>
+      <div className="flex flex-1 flex-col justify-between px-4 text-left">
+        <div className="flex flex-col">
+          <H3>{name}</H3>
+          <Prose>{role}</Prose>
+          <Prose>{institution}</Prose>
+          <a
+            className="text-uzh-red-100 hover:underline"
+            target="_blank"
+            href={link}
+            rel="noreferrer"
+          >
+            Profile
+          </a>
+        </div>
+        <div className="flex flex-col justify-end">
+          {tags.map((tag) => (
+            <div className="text-xs text-slate-700">{tag}</div>
```

Comment:
I would style them as actual "tags" (e.g., like on the game cards) :) Right now they look a bit lost and they do not use the space efficiently (go down, not in a row)